index.js

```diff
@@ -112,13 +141,33 @@ console.log(array1.fill(6));
     - Besides the methods on Person.prototype, babies have the ability to `.play()`:
         + Should return a string "Playing with x", x being the favorite toy.
 */
-function Baby() {
+const Baby=function (name,age,favoriteToy) {
   this.name=name;
   this.age=age;
+  this.favoriteToy=favoriteToy;
+  //this.play=function()[
+    //return 'Playing with x';
+  //]
+}
   
-
+this.play = function() {
+  return `Playing with ${this.favoriteToy}`
 }
 
+
+Baby.prototype = Object.create(Person.prototype);
+
+
+Baby.prototype.play = function() {
+return this.play;
+}	
+
```

Comment:
We needed to return a string containing its favorite toy like the instructions points out

> Should return a string "Playing with x", x being the favorite toy.

```js
Baby.prototype.play = function() {
  return "Playing with " + this.favoriteToy;
};